public/components/ctas/8.html

```diff
@@ -0,0 +1,9 @@
+<div class="p-20 bg-sky-50 flex items-center flex-wrap justify-between gap-y-8">
+		<div>
+			<p class="text-5xl font-semibold">Ready to grow?</p>
+			<p class="text-5xl font-semibold text-blue-700">Start your free trial now.</p>
+		</div>
+		<div>
+			<a href="#"><button class="py-4 px-6 text-xl text-slate-50 bg-blue-700 rounded-xl hover:bg-blue-600">Get Started</button></a>
```

Comment:
Why is there a `button` in an `a` tag? That's not how this HTML should be structured.

Can probably save some time by using one of the button components here.